src/ca11y/index.js

```diff
@@ -200,6 +222,7 @@ class Ca11y {
   }
 
   setUI(el) {
+    el.setAttribute('type', 'text') // in case this is [type="date"]
```

Comment:
Oh, right! This probably needs to be in master then, even if this feature is closed.